src/client.cpp

```diff
@@ -34,6 +34,7 @@ std::string zatPassword;
 bool zatFavoritesOnly = false;
 bool zatAlternativeEpgService = false;
 bool streamType = false;
+std::string xmlTVFile;
```

Comment:
No need to define the vars at the beginning of a function. This is not  C. In c++ it is good practice to  define variables where they are first used.